dash/src/blockdata/transaction/special_transaction/provider_update_registrar.rs

```diff
@@ -154,10 +155,10 @@ mod tests {
         assert_eq!(provider_mode, expected_provider_update_registrar_payload.provider_mode);
 
         let operator_key_hex = "139b654f0b1c031e1cf2b934c2d895178875cfe7c6a4f6758f02bc66eea7fc292d0040701acbe31f5e14a911cb061a2f";
-        assert_eq!(operator_key_hex, expected_provider_update_registrar_payload.operator_public_key.to_hex());
+        assert_eq!(operator_key_hex, expected_provider_update_registrar_payload.operator_public_key.encode_hex::<String>());
```

Comment:
```suggestion
        assert_eq!(operator_key_hex, hex::encode(expected_provider_update_registrar_payload.operator_public_key));
```

as we need no flexibility on varying types that could be built from char iterators

Same applies to other places in this PR

Reply:
I don't think what you said works @fominok

Reply:
why do you think so?

Reply:
works, I already applied this